### ISSUE TITLE
Adjust azure_lite autoScalerMax settings

### ISSUE THREAD

**Issue by PK85** [kyma-project/control-plane#1072]
**AC**

Adjust ```autoScalerMax``` for create and update on ```azure_lite``` plan to:
```
 "properties": {
                    "autoScalerMax": {
                      "default": 10,
                      "description": "Specifies the maximum number of virtual machines to create",
                      "maximum": 40,
                      "minimum": 2,
                      "type": "integer"
                    }
```

**Comment by mado0803** [comment]
@NHingerl is informed to keep the Documentation up to date once this is merged.

**Comment by NHingerl** [comment]
@PK85 I'd be aiming for publication on Dec 2nd. Will this work for you, or should I take precautions to postpone the update? (next date would be Dec 16th).
